features/fixtures/shared/scenarios/UserInfoScenario.swift

```diff
@@ -0,0 +1,47 @@
+//
+//  UserInfoScenario.swift
+//  iOSTestApp
+//
+//  Created by Jamie Lynch on 19/05/2020.
+//  Copyright © 2020 Bugsnag. All rights reserved.
+//
+
+import Foundation
+
+/**
+ * Sends a handled error to Bugsnag which  includes the default user information
+ */
+internal class UserInfoScenario: Scenario {
+
+    override func startBugsnag() {
+      self.config.autoTrackSessions = false;
+      super.startBugsnag()
+    }
+
+    override func run() {
+        let error = NSError(domain: "UserDefaultInfo", code: 100, userInfo: nil)
+        Bugsnag.notifyError(error)
+
+        after(.seconds(1)) {
```

Comment:
The scenario should try and avoid these waits if possible - it's going to lead to flaky behaviour if it takes more than 1s to call `Bugsnag.notify`. I'm not sure off the top of my head that it's required for bugsnag-cocoa - my guess would have been that the oldest error should always be delivered first (this could be wrong)?

Reply:
Agreed 👍 there's no need for these delays.

Errors will be delivered in the order of the calls to notify().

Reply:
Ah, ok - I had it in my mind that calls to notify were asynchronous. Perhaps they are, but there's still a queuing mechanism that means they will be delivered in order (or I'm just wrong about them being asynch!).

Reply:
No, calls to notify are synchronous.

Reply:
Delays removed